omnibus/package-scripts/chefdk/postrm

```diff
@@ -25,7 +25,7 @@ else
 fi
 
 cleanup_symlinks() {
-  binaries="chef chef-solo chef-apply chef-shell knife shef ohai berks chef-zero fauxhai foodcritic kitchen rubocop strain strainer chef-client"
```

Comment:
I don't think it will really matter, but which `postrm` gets executed on an upgrade? The one from the new package or the old? I _think_ this got left in here to clean up any `shef` bins when upgrading from 11.x -> 12.x. At this point there should probably not be many folks on a version of ChefDK with 11.x Chef Client, though, so the point may be moot.